src/applications/appeals/10182/content/additionalIssues.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+export const missingIssueErrorMessage = 'Please add the name of an issue';
+
+export const missingIssuesErrorMessage = (
+  <span className="usa-input-error-message" role="alert">
+    <span className="sr-only">Error</span>
+    Please add and select an issue, or select an eligible issue on the previous
+    page
+  </span>
+);
+
+export const AddIssuesIntroTitle = (
+  <p>
+    If the issue you were looking for was not in our system, you can add it
+    manually. You can only add an issue that you’ve already received a VA
+    decision notice for.
+  </p>
+);
+
+export const addIssuesIntroLabel =
+  'Would you like to add any additional issues for review?';
+
+export const AdditionalIssuesLabel = (
+  <span className="vads-u-font-weight--normal vads-u-font-size--base vads-u-font-family--sans">
+    Add an issue and our decision date on this issue. You can find the decision
```

Comment:
Should `our` be `your` in this sentence? Hard to tell.

Reply:
It should be `our` - it's information from a decision letter sent to the Veteran.